tap/passive_tapper.go

```diff
@@ -184,15 +177,15 @@ func (c *Context) GetCaptureInfo() gopacket.CaptureInfo {
 	return c.CaptureInfo
 }
 
-func StartPassiveTapper(opts *TapOpts) (<-chan *OutputChannelItem, <-chan *OutboundLink) {
+func StartPassiveTapper(opts *TapOpts, outputItems chan *api.OutputChannelItem, extensionsRef []*api.Extension, allExtensionPorts []string) {
 	hostMode = opts.HostMode
+	extensions = extensionsRef
 
-	harWriter := NewHarWriter(*HarOutputDir, *harEntriesPerFile)
-	outboundLinkWriter := NewOutboundLinkWriter()
```

Comment:
need to add comment on adding the outbound links after merging IMO (or maybe it will be done in another place)